ansible/tasks/deploy_react.yaml

```diff
@@ -6,8 +6,11 @@
     vars:
       IMAGE_TAG: '{{ version }}'
       deploy_params: "{{ lookup('template', './templates/deploy_react.param.j2') }}"
+  - debug: msg="Image Tag has not been created yet or the build is still in progress. A rollout has been triggered and may fail a few times before be successful"
+    when: not image_exists
```

Comment:
is this indenting correct?  I'm not that familiar with yaml, but should the conditional be at a lower indentation level than the logic?

Reply:
Yep its correct. That python indentation. Consider YAML more of JSON derivative. This is describing key/value pairs inside of an object
```json
[
 {
   "debug": "...",
   "when": "..."
 }
]
```

is equivalent